deltachat-ffi/deltachat.h

```diff
@@ -1447,6 +1447,55 @@ int             dc_set_chat_mute_duration             (dc_context_t* context, ui
 char*           dc_get_msg_info              (dc_context_t* context, uint32_t msg_id);
 
 
+/**
+ * Get uncut text of message, if available.
+ *
+ * Delta Chat tries to break the message in simple parts as plain text or images
+ * that are retrieved using dc_msg_get_type(), dc_msg_get_text(), dc_msg_get_file() and so on.
+ * This works totally fine for Delta Chat to Delta Chat communication,
+ * however, when the counterpart uses another E-Mail-client, this has limits:
+ *
+ * - even if we do some good job on removing quotes,
+ *   sometimes one needs to see them
+ * - HTML-only messages might lose information on conversion to text,
+ *   esp. when there are lots of embedded images
+ * - even if there is some plain text part for a HTML-message,
+ *   this is often poor and not nicely usable due to long links
+ *
+ * In these cases, dc_msg_is_mime_modified() returns 1
+ * and you can ask dc_get_original_mime_html() for some HTML-code
+ * that shows the uncut text (which is close to the original)
+ * For simplicity, the function _always_ returns HTML-code,
+ * this removes the need for the UI
+ * to deal with different formatting options of PLAIN-parts.
+ *
+ * **Note:** As usual for HTML,
```

Comment:
```suggestion
 * **Note:** As usual for user-generated HTML,
```

Reply:
Maybe just drop "as usual for HTML,"?

Reply:
yip, i also think, we can drop that completely instead for further refining.